source/content/guides/wordpress-composer/03-create-wp-site-composer-ci-auto-test.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Create an Advanced Composer-managed WordPress Site 
+subtitle: Create an Advanced Composer-managed WordPress Site 
+description: Learn how to create a WordPress Site that uses Composer, Continuous Integration, and Automated Testing on Pantheon
+categories: [develop]
+tags: [wordpress]
+contributors: [whitneymeredith]
+layout: guide
+showtoc: true
+permalink: docs/guides/wordpress-composer/create-wp-site-composer-ci-auto-test
+anchorid: create-wp-site-composer-ci-auto-test
+---
+
+This section provides steps to create a new Pantheon WordPress site that will use [Composer](https://getcomposer.org/), Continuous Integration, and Automated Testing. This guide will get you started, but you will need to customize and maintain the CI/testing set up for your projects.
+
+## Requirements
+
+- [Terminus Build Tools Plugin](https://github.com/pantheon-systems/terminus-build-tools-plugin)
+
+    - You must use the [Build Tools 3.x release](https://github.com/pantheon-systems/terminus-build-tools-plugin/tree/3.x) if you are using **Terminus 3**.
+
+    - You must use the [Build Tools 2.x release](https://github.com/pantheon-systems/terminus-build-tools-plugin/tree/2.x) if you are using **Terminus 2**.
+
+- [PHP version](/php-versions#verify-current-php-versions) 7.2 or greater
+
+## Create Your Site
+
+1. Use the [Terminus Build Tools Plugin](https://github.com/pantheon-systems/terminus-build-tools-plugin) to copy the Pantheon-maintained [WordPress Composer repository](https://github.com/pantheon-systems/example-wordpress-composer).
```

Comment:
There's no "how" in this guide for this alternative.

`terminus build:project:create wp` should be used as a very basic, but several options are available

Reply:
Note that using build tools will spin up the site based on https://github.com/pantheon-systems/example-wordpress-composer

Reply:
Note that using build tools will spin up the site based on https://github.com/pantheon-systems/example-wordpress-composer